.github/workflows/fetch-related-content.yml

```diff
@@ -47,50 +49,12 @@ jobs:
           git commit -m 'chore(related-content): updated related content data'
           echo "::set-output name=commit::true"
 
-      - name: Temporarily disable "required_pull_request_reviews" branch protection
```

Comment:
Removed these because we do not currently have branch protection on the main branch.